### PR TITLE
Additional of EncodePNG method alongside SavePNG to allow for extract…

### PR DESCRIPTION
**Pull request by johnha**
Addition of of the binary image extract (EncodePNG) in addition to the current file save.   Added test case and example (example 12).

### CODE REVIEW

**Comment by shomali11** [comment]
Thank you for this! Could you remove the temporary change and I will merge it

**Comment by johnha** [comment]
Done.

Thanks for library - very nice!

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/shomali11/gridder
+module github.com/johnha/gridder
```
Comment (shomali11) [inline go.mod:1]: I still see this.

**Comment by shomali11** [comment]
I am glad you like the library. It’s my favorite!

**Comment by johnha** [comment]
ok - see my mistake - I just undid the merge on my fork - I have updated the .mod on that branch.

I love the simplicity of it.   Trying to include as a library in startlark…  (embedded python interpreter).   Will see how it goes…

cheers

**Pull request merged**